AEPCore/Sources/eventhub/ExtensionRuntime.swift

```diff
@@ -93,4 +93,12 @@ public protocol ExtensionRuntime {
     ///   - event: If not nil, will retrieve the `SharedState` that corresponds with the event's version, if nil will return the latest `SharedState`
     /// - Returns: A `SharedStateResult?` for the requested `extensionName` and `event`
     func getXDMSharedState(extensionName: String, event: Event?) -> SharedStateResult?
```

Comment:
can we just add an optional parameter to this method instead of adding a new method?

Reply:
yep, updated.